portfolio/src/main/webapp/projects.html

```diff
@@ -121,7 +118,12 @@ <h4>Computer Science Fundamentals Website</h4>
                 </p>
             </li>
         </ul>
-      </p>
+    </p>
+    <h2>Google Charts Library</h2>
+    <p>Graphs made from the Google Charts Library</p>
+    <div id="chart-container"></div>
+    <div id="chart-csv-container"></div>
+    <p>Resource: http://www.ecdms.energy.ca.gov/elecbycounty.aspx</a></p>

```

Comment:
I don't see a matching opening tag for <a>. Did you want to make the URL a hyperlink?

Reply:
Added extra opening tag.